src/components/body/annotation.js

```diff
@@ -49,8 +51,8 @@ const Indicator = styled.span`
   margin-left: 3px;
   display: inline-block;
   vertical-align: middle;
-  width: 1em;
-  height: 1em;
+  width: 1rem;
+  height: 1rem;
```

Comment:
Since the strokes of the arrow is fixed with unit `px`, we can change circle size to fixed `18px` (the `rem` of user browser may differ)